src/styles/typography.css

```
@@ -81,18 +82,18 @@
-      @apply rounded bg-muted/75 p-1 break-words font-mono text-foreground before:content-none after:content-none;
+      @apply rounded bg-muted/75 p-1 font-mono break-words text-foreground before:content-none after:content-none;
     }
 
     .astro-code code {
       @apply flex-[1_0_100%] bg-inherit p-0;
     }
 
     blockquote {
-      @apply border-s-accent/80 break-words font-sans opacity-80;
+      @apply border-s-accent/80 font-sans break-words opacity-80;
       quotes: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)